src/group/components/GroupView.test.js

```diff
@@ -98,20 +115,15 @@ test('GroupView: Display data', async () => {
         },
       })),
   };
-  const accountMembership = {
-    id: 'user-id',
-    userRole: 'member',
-    membershipStatus: 'APPROVED',
-    user: {
-      firstName: 'Member First Name',
-      lastName: 'Member Last Name',
-    },
-  };
+
   const sharedResources = {
     edges: Array(6)
       .fill(0)
       .map((item, index) => ({
         node: {
+          id: `shared-resource-id-${index}`,
+          shareAccess: 'TARGET_MEMBERS',
+          shareUrl: 'https://test-url',
```

Comment:
question: should this be a valid URL?
```suggestion
          shareUrl: 'https://example.com,
```